src/compressor/brotli/CMakeLists.txt

```diff
@@ -8,12 +8,12 @@ include(ExternalProject)
 ExternalProject_Add(brotli_ext
   DOWNLOAD_DIR ${CMAKE_BINARY_DIR}/src/
   GIT_REPOSITORY "https://github.com/google/brotli.git"
-  GIT_TAG "master"
-  SOURCE_DIR ${CMAKE_BINARY_DIR}/src/brotli
-  CONFIGURE_COMMAND ./configure-cmake --disable-debug 
+  GIT_TAG "v1.0.7"
+  SOURCE_DIR ${CMAKE_BINARY_DIR}/src//brotli
+  CONFIGURE_COMMAND ${CMAKE_BINARY_DIR}/src/brotli/configure-cmake --disable-debug 
```

Comment:
why do you trade `BUILD_IN_SOURCE` with the absolute path?

Reply:
Because of the previous method of downloading the file to the source folder, I change using out-of-source build method to build. Now, I have been changed back to the BUILD_IN_SOURCE.